tests/executors/test_kubernetes_executor.py

```diff
@@ -1253,6 +1253,21 @@ def test_supports_pickling(self):
     def test_supports_sentry(self):
         assert not KubernetesExecutor.supports_sentry
 
+    def test_annotations_to_str(self):
+        executor = self.kubernetes_executor
+        executor.scheduler_job_id = "modified"
```

Comment:
```suggestion
```

Reply:
Yeah these are not required, made the required changes